lib/core/test/unit/Cardano/Wallet/DB/Sqlite/StoresSpec.hs

```diff
@@ -185,3 +295,13 @@ dummyUTCTime = posixSecondsToUTCTime 1506203091
 dummyHash :: Hash "BlockHeader"
 dummyHash = Hash $ unsafeFromHex
     "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb"
+
+{-------------------------------------------------------------------------------
```

Comment:
what about relocating this function for example here https://github.com/input-output-hk/cardano-wallet/blob/master/lib/test-utils/src/Test/QuickCheck/Extra.hs It seems to be very useful